test/data/core_config.json

```diff
@@ -72,7 +72,8 @@
 		"maxTransactionsPerQueue": 1000
 	},
 	"forging": {
-		"force": false,
+		"force": true,
```

Comment:
Why `force` to `true` and `defaultPassword`? It will not work, as `delegates` array is empty.